BUILDING-configure.md

```diff
@@ -1,26 +1,36 @@
 Building using `configure`
 ==========================
 
-The build requires `libpq`, the C client library for PostgreSQL.  This library
-must be installed before you can build libpqxx.  You'll need the headers as
-well as the library binary.
-
 The instructions will assume that you're working from a command-line shell.
 If you prefer to work from an IDE, you'll have to know how your IDE likes to
 do things, and you'll want to follow the shell instructions as a guide.
 
 
+Prerequisites
+-------------
+
+Before attempting to install the `libpqxx` library, you first need to 
```

Comment:
I think there's a bit of room for confusion here, still: it's fine if the reader understands "install" to mean "going through this entire process."  But what if they just want to do a build, for instance?  Then "install" means just that final `make install` step.  And you need libpq before the _building_ part.

One way to address that might be to say "Before attempting to build and install" instead of just "Before attempting to install".

(And: TBH the "attempting" sounds a bit corporate and also pessimistic...  Perhaps we can leave that word out?)

Reply:
Oh, let me look at this real quick.

Reply:
Ok, change is made.